src/Operation/DropCollection.php

```diff
@@ -19,6 +19,7 @@
 
 use MongoDB\Driver\Command;
 use MongoDB\Driver\Exception\CommandException;
+use MongoDB\Driver\Exception\RuntimeException as DriverRuntimeException;
```

Comment:
Interesting that `phpcbf` didn't catch this. It's only referenced from a `@throws` annotation.

Reply:
I don't think phpcbf validates that `@throws` references a valid class. It probably cannot do it, as it works only on a single file IIRC.